germanium/test_cases/default.py

```diff
@@ -3,9 +3,13 @@
 from django.conf import settings
 from django.test.testcases import TestCase, SimpleTestCase
 
+from germanium.config import MILTI_DB_TESTS
+
 
 class GermaniumSimpleTestCaseMixin:
 
+    multi_db = MILTI_DB_TESTS
```

Comment:
this parameter is removed in django 3.0 and is undocumented :(

Reply:
imho it's only deprecated and there is a replacement 🤷‍♂

Reply:
django 3.0 it just remove it means that

* default value for django<3.0 is false
* default and only value for django >= 3.0 is true